docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,171 @@
-
-# Overview
-
-Just like smart contracts, Smart Rollups are decentralized software
-components. However, contrary to smart contracts that are processed by
-the network validators automatically, a smart rollup requires a
-dedicated *rollup node* to function.
-
-Any user can originate, operate, and interact with a rollup. For the
-sake of clarity, we will distinguish three kinds of users in this
-documentation: operators, kernel developers, and end-users. An operator
-deploys the rollup node to make the rollup progress. A kernel developer
-writes a kernel to be executed within a rollup. An end-user interacts
-with the rollup through layer 1 operations or Layer 2 input messages.
-
-## Address
-
-When a smart rollup is originated on layer 1, a unique address is
-generated to uniquely identify it. A smart rollup address starts with
-the prefix `sr1`.
-
-## Inputs
-
-There are two channels of communication to interact with Smart Rollups:
-
-1.  a global **rollups inbox** allows layer 1 to transmit
-    information to all the rollups. This unique inbox contains two kinds
-    of messages: *external* messages are pushed through a layer 1
-    manager operation while *internal* messages are pushed by layer 1
-    smart contracts or the protocol itself.
-2.  a **reveal data channel** allows the rollup to retrieve data coming
-    from data sources external to layer 1.
-
-### External messages
-
-Anyone can push a message to the rollups inbox. This message is a mere
-sequence of bytes following no particular underlying format. The
-interpretation of this sequence of bytes is the responsibility of each
-kernel.
-
-There are two ways for end-users to push an external message to the
-rollups inbox: first, they can inject the dedicated layer 1 operation
-using the Octez client second, they can use
-the batcher of a smart rollup node (see [Sending an External Inbox Message](#sending-an-external-inbox-message)).
-
-### Internal messages
-
-Contrary to external messages, which are submitted by the end users,
-internal messages are constructed by layer 1.
-
-At the beginning of every Tezos block, layer 1 pushes two internal
-messages:
-
-- `"Start of level"` - no associated payload
-- `"Info per level"` - provides
-to the kernel the timestamp and block hash of the predecessor of the
-current Tezos block.
-
-A rollup is identified by an address and has an associated Michelson
-type (defined at origination time). Any layer 1 smart contract can
-perform a transfer to this address with a payload of this type. This
-transfer is realized as an internal message pushed to the rollups inbox.
-
-Finally, after the application of the operations of the Tezos block, the layer 1 pushes one final internal message `"End of level"`. Similarly to `"Start of level"`, this internal messages does not come with any payload.
+### Rollup inbox
+
+As part of its state, layer 1 has a _rollup inbox_ that contains messages from layer 1 to all rollups.
```

Comment:
I changed this to "as part of its state" because we wanted to describe the rollup inbox not as part of any individual block but as part of the L1 state.

Reply:
What I'm saying is precisely that this is not correct. The L1 state (aka the L1 "context") contains things such as contract balances; but it does not contain the L2 inbox, AFAIU.

Reply:
Asked for clarification in a smart rollups slack channel.

Reply:
Per the smart rollups channel, it's more technically accurate to say that each layer 1 block has a smart rollup inbox. I'll change this to "Each layer 1 block contains a _rollup inbox_..."